pkg/network/network.go

```diff
@@ -25,7 +26,13 @@ var templates embed.FS
 // ReconcileNetwork reconciles the network settings for a firewall
 // in the current stage it only changes the FRR-Configuration when network prefixes or FRR template changes
 func ReconcileNetwork(f firewallv1.Firewall, log logr.Logger) (bool, error) {
-	kb := netconf.NewKnowledgeBase(MetalKnowledgeBase)
+	// FIXME use zapr ?
+	zlog, _ := zap.NewProduction()
+
+	kb, err := netconf.New(zlog.Sugar(), MetalKnowledgeBase)
```

Comment:
`MetalKnowledgeBase` -- should we rename it to `MetalNetworkerConfig`?

Reply:
Yes

Reply:
done